tests/test_dse.py

```diff
@@ -1716,44 +1717,6 @@ def test_blocking_options(self, rotate):
         assert np.isclose(expected, norm(u1), rtol=1e-5)
         assert np.isclose(expected, norm(u2), rtol=1e-5)
 
-    @pytest.mark.parametrize('rotate', [False, True])
-    def test_arrays_enforced_on_stack(self, rotate):
```

Comment:
Dropped as now the tensor temporaries can only be allocated on the heap (stack was flaky and performance-wise useless, so why keep it?)